specification/cognitiveservices/OpenAI.Inference/models/completions/azure_chat_extensions.tsp

```diff
@@ -94,6 +99,51 @@ model AzureChatExtensionDataSourceResponseCitation {
   chunk_id?: string;
 }
 
+/** The retrieved document. */
+@added(ServiceApiVersions.v2024_05_01_Preview)
+model AzureChatExtensionRetrievedDocument {
+  ...AzureChatExtensionDataSourceResponseCitation;
+
+  /** The search queries used to retrieve the document. */
+  @encodedName("application/json", "search_queries")
+  searchQueries: string[];
+
+  /** The index of the data source. */
+  @encodedName("application/json", "data_source_index")
+  dataSourceIndex: int32;
+
+  /** The original search score of the retrieved document. */
+  @encodedName("application/json", "original_search_score")
+  originalSearchScore?: float32;
```

Comment:
The significance is unclear, but the spec specifically uses the "double" format for this and the following rerank score, which I believe implies this should be `float64` (double vs. float precision)